neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/remote/http/impl/HttpNeo4jSchemaDefiner.java

```diff
@@ -53,6 +54,19 @@ protected void createSequences(List<Sequence> sequences, Set<IdSourceKeyMetadata
 		sequenceGenerator.createSequences( sequences, allIdSourceKeyMetadata );
 	}
 
+	@Override
+	protected void createIndexesIfMissing(DatastoreProvider provider, List<Neo4jIndexSpec> indexes) {
+		Statements statements = new Statements();
+		for ( Neo4jIndexSpec index : indexes ) {
+			log.tracef( "Creating composite index for nodes labeled as %1$s on properties %2$s", index.getLabel(), index.getProperties() );
+			statements.addStatement( new Statement( index.asCypherQuery() ) );
+		}
+		log.debug( "Creating missing indexes" );
+		HttpNeo4jClient remoteClient = ( (HttpNeo4jDatastoreProvider) provider ).getClient();
```

Comment:
When we close the instance of HttpNeo4jClient?

Reply:
I think it's when we close the `HttpNeo4jDatastoreProvider`

Reply:
you're right :P